composer.json

```
@@ -5,16 +5,16 @@
         "psr-4": { "M6Web\\Bundle\\": "src/" }
     },
     "require" : {
-        "php": "~7",
+        "php": ">=7.2",
```

```suggestion
        "php": ">=7.4",
```

since we're doing a breaking change by dropping php7.0 and php7.1, we could also drop other unsupported PHP versions, WDYT?